notes.py

```diff
@@ -384,6 +424,18 @@ def sort_guitar_positions(p: list[GuitarPosition], target_fret: int = 7) -> list
     ))
 
 
+def filter_subset_guitar_positions(p: list[GuitarPosition]) -> list[GuitarPosition]:
```

Comment:
When we allow repeated notes, we get a bunch of extra chords that have various subsets of the strings played. This isn't really helpful, ideally we would just pick the largest one that is playable.